uikit/positionlockfab/viewmodel/src/main/java/com/sygic/ui/viewmodel/positionlockfab/PositionLockFabViewModel.kt

```diff
@@ -48,7 +55,7 @@ class PositionLockFabViewModel internal constructor(
         modeChanged()
     }
 
-    private fun modeChanged() {
+    protected fun modeChanged() {
```

Comment:
toto je taka divna metoda, ze hovori, ze sa nieco zmenilo, ale nehovori co. Ked to bolo private, tak ok, vedeli sme my, o co ide. Ale ako protected to nie je moc dobre zapisane

Reply:
pravdu máš, nechám ji private, může si override onRotationModeChanged nebo onMovementModeChanged, ze kterých se volá ;)